docs/source/getting_started/platform.rst

```diff
@@ -194,13 +194,77 @@ need to look like this:
         ]
     }
 
+If you don't have your data in coco format yet, but e.g. as a pandas dataframe,
+you can use a simple script to translate it to the coco format:
+
+.. code-block:: python
+
+    ## OPTION 1: pandas dataframe with named columns
+    # This works well if you have many columns and column names
+    import pandas as pd
+    from lightly.utils import save_custom_metadata
+
+    # Define the pandas dataframe
+    column_names = ["filename", "number_of_pedestrians", "scenario", "temperature"]
+    rows = [
+        ["image0.jpg", 3, "cloudy", 20.3],
+        ["image1.jpg", 1, "rainy", 15.0]
+    ]
+    df = pd.DataFrame(rows, columns=column_names)
+
+    # create a list of pairs of (filename, metadata)
+    # metadata must be a dictionary
```

Comment:
I liked it better before, why not simply do this:

```python
custom_metadta = []
for row in df.itertuples():
    filename = data.filename
    metadata = {
        "number_of_pedestrians": int(data.number_of_pedestrians),
        "weather": {
             "scenario": str(data.scenario),
             "temperature": float(data.temperature),
        }
    }
    custom_metadata.append((filename, metadata))
```

This is easy to read and allows for all the nesting we want.

Reply:
I followed your suggestion and used
- the named dataframe from option 1
- the named and nested metadata creation you just proposed